dev-infrastructure/modules/aks-cluster-base.bicep

```diff
@@ -316,7 +316,6 @@ resource aksCluster 'Microsoft.ContainerService/managedClusters@2024-04-02-previ
         mode: 'System'
         enableAutoScaling: true
         enableEncryptionAtHost: true
-        enableFIPS: true
```

Comment:
We should leave this alone as it's for infrastructure, not the HCP OpenShift cluster resources.